.lintstagedrc.json

```diff
@@ -1,4 +1,5 @@
 {
-  "**/*.{mjs,ts,tsx,md,mdx}": ["eslint --fix"],
-  "**/*.{mjs,ts,tsx,md,mdx,json,yml,css,sass,scss}": ["prettier --write"]
+  "**/*.{mjs,ts,tsx,md,mdx}": ["prettier --write", "eslint --fix"],
+  "**/*.{json,yml}": ["prettier --write"],
+  "**/*.{css,sass,scss}": ["prettier --write", "stylelint --config .stylelintrc.json"],
```

Comment:
I'm not sure if the config with this relative path will work, to be honest. These changes need to be tested locally.